src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -199,6 +199,7 @@ public static class Release
                 public static readonly string ReleaseWebUrl = "release.releaseWebUrl";
                 public static readonly string RequestorId = "release.requestedFor";
                 public static readonly string SkipArtifactsDownload = "release.skipartifactsDownload";
+                public static readonly string ReleaseDefinitionId = "release.definitionId";
```

Comment:
Can you fix the order of this property? Everything else in the file is alphabetical.